csfieldguide/static/interactives/city-trip/README.md

```diff
@@ -0,0 +1,5 @@
+# City Trip Interactive
+
+**Author:** Courtney Bracefield
+
+This interactive is created to demonstrate the travelling salesman problem.
```

Comment:
I think you could flesh out both of these a bit - How does it demonstrate TSP? What can the user do? What third-party libraries were used here and how can I find their licences?

Reply:
Also this interactive existed before you recreated it, so mention the original author somewhere